uat/terraform/vpc/vpc.tf

```diff
@@ -0,0 +1,30 @@
+# AWS Virtual Private Cloud
+
+## Variables
+variable "enable_dns" {}
+
+variable "enable_hostnames" {}
+
+variable "region" {}
+
+variable "stack_item_fullname" {}
+
+variable "stack_item_label" {}
+
+variable "vpc_cidr" {}
+
+## Configures AWS provider
+provider "aws" {
+  region = "${var.region}"
+}
+
+## Configures base VPC
+module "vpc_base" {
```

Comment:
hmmm for testing i was thinking we could probably use something like ec2-classic without an VPC since it has unlimited resource limits (well, much higher default limits 200+ vs 5). I'll fiddle with this a little bit in covalence.